keps/sig-etcd/4578-feature-gate/kep.yaml

```diff
@@ -0,0 +1,18 @@
+title: etcd Feature Gate
```

Comment:
This should match the title for issue 4578 (currently: “Introduce feature gate for etcd”).

Reply:
updated. Thanks!